solvers/vu_condat_cd.py

```diff
@@ -0,0 +1,56 @@
+from benchopt import BaseSolver
+from benchopt import safe_import_context
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from cd_solver.sklearn_api import Lasso
+
+
+class Solver(BaseSolver):
+    name = 'vu-condat-cd'
+
+    install_cmd = 'conda'
+    requirements = [
+        'pip:git+https://github.com/Badr-MOUFAD/Fercoq-Bianchi-solver.git@sklearn-api'
```

Comment:
To discuss: wouldn't it be more rigorous to add it from the bitbucket repository of @ofercoq directly? We don't really need the sklearn API, just a call to the solver.
Also installing from a branch is fragile, as it could be deleted in the future.

Reply:
100% Agree!

I will unwrap the logic of the Lasso inside the solver itself.